include/exadg/postprocessor/time_control.cpp

```diff
@@ -0,0 +1,173 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+// C/C++
+#include <limits>
+
+// deal.II
+#include <deal.II/base/exceptions.h>
+
+// ExaDG
+#include <exadg/postprocessor/time_control.h>
+#include <exadg/utilities/print_functions.h>
+
+namespace ExaDG
+{
+TimeControlData::EvalType
+get_unsteady_evaluation_type(TimeControlData const & data)
+{
+  bool const interval = (data.trigger_interval > 0.0);
+  bool const timestep = Utilities::is_valid_timestep(data.trigger_every_time_steps);
+
+  // only one of both can be true
```

Comment:
This comment irritates me, because of "can". A user can specify both variables in "application.h". I would better say "We do not allow users to specify both a time interval and a number of time steps."